kymatio/scattering1d/frontend/keras_frontend.py

```diff
@@ -1,26 +1,21 @@
 from ...frontend.keras_frontend import ScatteringKeras
-from ...scattering1d.frontend.base_frontend import ScatteringBase1D
+from ...scattering1d.frontend.base_frontend import ScatteringBase1D, TimeFrequencyScatteringBase
 
 from kymatio.tensorflow import Scattering1D as ScatteringTensorFlow1D
+from kymatio.tensorflow import TimeFrequencyScattering as TimeFrequencyScatteringTensorFlow
 
 from tensorflow.python.framework import tensor_shape
 
 
 class ScatteringKeras1D(ScatteringKeras, ScatteringBase1D):
     def __init__(self, J, Q=1, T=None, max_order=2, oversampling=0):
         ScatteringKeras.__init__(self)
-        self.J = J
-        self._Q = Q
-        self._T = T
-        self._max_order = 2
-        self._oversampling = 0
-        self.out_type = 'array'
-        self.backend = None
+        ScatteringBase1D.__init__(self, J, None, Q=Q, T=T, max_order=2, oversampling=0, out_type='array')
```

Comment:
yes. this is not necessary for JTFS but nice-to-have and it improves consistency across frontends